ui/src/app/app-routing.module.ts

```diff
@@ -14,10 +14,12 @@ import {SelectRepositoryComponent} from 'app/views/select-repository/component';
 
 
 const routes: Routes = [
-  {path: '', component: HomePageComponent, data: {title: 'All of Us Researcher Workbench'}},
+  {path: '', component: HomePageComponent, data: {title: 'View Workspaces'}},
```

Comment:
Maybe just "Workspaces" and "Workspace Details." I think of a title as more of a noun than a verb.